lms/static/sass/shared/_header.scss

```diff
@@ -265,11 +265,13 @@ header.global {
     .tab-nav-item{
       display: inline;
       margin: 0px;
+      text-transform: none;
       .tab-nav-link{
         font-size: em(16);
         color: $gray;
-        padding: 26px 15px;
-        display: inline;
+        padding: 5px 25px 23px;
+        display: inline-block;
+        margin-left: -4px;
```

Comment:
As discussed offline it would be great to leverage flexbox instead of having to use negative margins to resolve the issue with inline-block elements adding extra spacing.